### PR TITLE
toSnakeCase function

### PR DESCRIPTION
**Pull request by dovbysh**

### CODE REVIEW

File: cmd_generate.go [hunk 1]
```diff
@@ -255,11 +256,12 @@ func varsToArgs(v vars) string {
...
-	"down":      strings.ToLower,
-	"upFirst":   upFirst,
-	"downFirst": downFirst,
-	"replace":   strings.ReplaceAll,
+	"up":          strings.ToUpper,
+	"down":        strings.ToLower,
+	"upFirst":     upFirst,
+	"downFirst":   downFirst,
+	"replace":     strings.ReplaceAll,
+	"toSnakeCase": toSnakeCase,
```
Comment (hexdigest) [inline cmd_generate.go:264]: @dovbysh Can you please change the key to "snake" just to make it more useful within a templates?

**Review by hexdigest — CHANGES_REQUESTED** [review]

File: cmd_generate.go (hunk 1 above)
Comment (AntonRutkevich) [reply cmd_generate.go:264]: @dovbysh wow, I was just going to implement this same function) Could you please fix Max's comment and merge the PR?

Comment (dovbysh) [reply cmd_generate.go]: Да, конечно. Времени не хватало.

**Comment by hexdigest** [comment]
@dovbysh Andrey, thanks for your contribution!
This functionality is available in v1.1.11

cc @AntonRutkevich

**Comment by AntonRutkevich** [comment]
Thanks!

**Pull request merged**